### ISSUE TITLE
Choppy audio and bad framerate in games

### ISSUE THREAD

**Issue by faegliss** [rafaelvcaetano/melonDS-android#921]
* **Device model:** Huawei Y6 2019
* **Android version:** 9

**Problem:**  
Extremely choppy audio in almost all of my games, not to mention the framerate.
Extremely slow and overall choppy gameplay.

**Steps to reproduce (if applicable):**
1. I open a game of my choice
2. The audio is glitchy and choppy
3. And framerate during gameplay also decreases

Add screenshots if the issue can be visualized.
https://user-images.githubusercontent.com/103389369/170861893-1c8b292f-9c6c-4126-8beb-4dd9f12952aa.mp4

**Comment by faegliss** [comment]
also, sorry if the video can't be heard well, I had my volume turned down because it was hell to listen to the audio

also, I don't know why the screenshot appears like a link, sorry about that.
As far as I am concerned, I believe DS games can run on android 9.

**Comment by cyruzwings** [comment]
the video shown is lemuroid with melonds core?

**Comment by faegliss** [comment]
> the video shown is lemuroid with melonds core?
> 

Yes, can we only report issues in the MelonDS standalone app ? Sorry if thats the case :')

**Comment by rafaelvcaetano** [comment]
> Yes, can we only report issues in the MelonDS standalone app ? Sorry if thats the case :')

Since this is the repository for the standalone melonDS version, it makes sense to only report issues from that app here. You can report Lemuroid issues on its repository: https://github.com/Swordfish90/Lemuroid

**Comment by faegliss** [comment]
> > Yes, can we only report issues in the MelonDS standalone app ? Sorry if thats the case :')
> 
> Since this is the repository for the standalone melonDS version, it makes sense to only report issues from that app here. You can report Lemuroid issues on its repository: https://github.com/Swordfish90/Lemuroid

ah yeah, sorry :')
i opened this issue a while ago, looking back it was pretty stupid to report the issue here rather than the lemuroid repo.
again, soery for that :)
